client/src/components/home/HomePage.tsx

```diff
@@ -97,11 +98,16 @@ export default class HomePage extends Component<IHomePageProps, IHomePageState>
   };
 
   public getValidationState = () => {
```

Comment:
_I wrote a comment here earlier, but I forgot to add it_

There is no need to return a dictionary with 2 variables, because, actually, `disabled` is tightly coupled with `validation`. It's easier to follow the logic in this component if you simply return just the `Validation` type. Don't forget to adapt to the change in `#render` method accordingly.